airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/config_migrations.py

```diff
@@ -121,3 +121,76 @@ def transform(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
             config[stream_filter] = statuses
         # return transformed config
         return config
+
+
+class MigrateSecretsPathInConnector:
+    """
+    This class stands for migrating the config at runtime.
+    This migration is intended for backwards compatibility with the previous version, so existing secrets configurations gets migrated to new path.
+
+    Starting from `3.0.0`, the `client_id`, `client_secret` and `access_token` will be placed at `credentials` path.
+    """
+
+    message_repository: MessageRepository = InMemoryMessageRepository()
```

Comment:
This field is useless right now. We can just do `print(create_connector_config_control_message(migrated_config).json(exclude_unset=True))`